src/domains/ecommerce-common/03_storage.tf

```diff
@@ -160,6 +160,11 @@ resource "azurerm_storage_queue" "transactions_notifications_queue_blue" {
 
 
 
+resource "azurerm_storage_queue" "transactions_authorization_requested_queue" {
```

Comment:
please add also queue for blue deployment like done [here](https://github.com/pagopa/pagopa-infra/blob/04f610e15918617e6c798d2e5778cca9c17a2c21/src/domains/ecommerce-common/03_storage.tf#L125-L129)

Reply:
Solved in 9aada0a